packages/valaxy/client/locales/en.yml

```diff
@@ -1,14 +1,14 @@
 button:
   about: About
   back: Back
-  go: GO
+  go: Go
   home: Home
-  toggle_dark: Toggle dark mode
-  toggle_langs: Change languages
+  toggle_theme: Toggle theme
```

Comment:
```yml
button:
  toggle_light: Toggle light mode
  toggle_dark: Toggle dark mode
```

And judge it by `isDark`.

https://github.com/YunYouJun/valaxy/blob/10a4a9e92afaf139be5d18c2a6456a58b1063fb2/packages/valaxy-theme-yun/components/YunConfig.vue#L10

Reply:
No, `isDark` sounds good to me.

Reply:
Right, that sounds cool.

Reply:
That's OK.

Reply:
Updated